xonsh/readline_shell.py

```diff
@@ -204,9 +204,7 @@ def _cmdloop(self, intro=None):
                     if inserter is not None:
                         readline.set_pre_input_hook(None)
                 else:
-                    self.stdout.write(self.prompt.replace('\001', '')
-                                                 .replace('\002', ''))
-                    self.stdout.flush()
+                    print_color(self.prompt)
```

Comment:
This does not exactly replicate the code that was there originally.  In particular, the original uses `self.stdout`, while the `print_color()` function will go to `sys.stdout`.